### ISSUE TITLE
All blog links are dead?

### ISSUE THREAD

**Issue by pombredanne** [jruizgit/rules#204]
All the blog links seem to be dead http://jruizblog.com/2015/07/20/miss-manners-and-waltzdb/ ... though they  likely quite valuable insights.

**Comment by pombredanne** [comment]
Actually https://web.archive.org/web/*/http://jruizblog.com/ has them still

**Comment by simkimsia** [comment]
@jruizgit I agree with @pombredanne observation that the blog links appear to be dead. Can advise?

**Comment by vduseev** [comment]
@simkimsia @pombredanne 
I think blog is still accessible and was moved to the https://jruizblog.wordpress.com/

**Comment by simkimsia** [comment]
Copy that thank you
